app/views/facilities/_quicklinks.html.haml

```diff
@@ -0,0 +1,21 @@
+.quick_links 
+  .add_to_cart
+    - if not session_user
+      = link_to "#{t("admin.shared.add_to",model:t("pages.cart"))} (#{t("pages.login")})", new_user_session_path
+    - elsif session_user.can_override_restrictions?(product) || product.can_be_used_by?(acting_user)
+      - if not session_user.accounts.empty?
+        = link_to t("admin.shared.add_to",model:t("pages.cart")), add_order_path(acting_user.cart(session_user), order: {order_details: [{product_id: product.id, quantity: 1}] }), method: :put
+      - else
+        = t("facility_accounts.account_table.notice")
+    - else
+      = t("facility_account_orders.cannot_purchase")
+  %li{class: [product.class.model_name.to_s.downcase, 'view_product']}
+    = public_calendar_link(product, indent: indent_for_icon)
+    = link_to 'View' + (product.is_hidden? ? ' (hidden)' : ''), facility_product_path(current_facility || product.facility, product), :id => product.name
+    - if acting_user.present? && !product.can_be_used_by?(acting_user)
+      %i.fa.fa-lock
+      = " (#{product.class.human_attribute_name(:requires_approval_show)})"
+
+      - if product.offline?
```

Comment:
Not sure why, but this isn't rendering for me.

Reply:
How can we replicate it? I'm using the existing logic, but we don't have a locked facility to test with. If we could set it up the same way we can fix it.

Reply:
You can mark an instrument offline by going into the products tab from manage facilities, choose an instrument. You should be taken to the "Reservations". One of the buttons should be "Mark Instrument Offline". Put in a note and a category and the instrument will be considered offline. This feature is meant for unplanned maintenance and issues.

Once you've marked it offline, on the product listing on the "use" side, you should see a red triangle with an exclamation mark.

Reply:
Looks like the indentation is wrong. This is nested in the `- if acting_user.present? && !product.can_be_used_by?(acting_user)`. It should be a separate piece of logic.